app/auth-portal/src/App.vue

```diff
@@ -103,7 +103,7 @@
               variant="transparent"
               tone="shade"
               @mousedown.prevent
-              @click.prevent="profileMenuRef?.open"
+              @click.prevent="profileMenuRef?.open || _.noop"
```

Comment:
shouldnt really need this but it was causing a vue internals bug... I expect it to be fixed pretty quickly.

https://github.com/vuejs/core/issues/9865

Reply:
looks like the issue itself is resolved but we'll need to wait for a few dependencies to update to the latest vue (not just us)